integration/test/alert/alert.go

```diff
@@ -1,55 +1,133 @@
 package promAlerts
 
 import (
+	"fmt"
 	"github.com/Netapp/harvest-automation/test/utils"
+	"github.com/netapp/harvest/v2/pkg/conf"
 	"github.com/netapp/harvest/v2/pkg/tree"
 	"github.com/rs/zerolog/log"
 	"github.com/tidwall/gjson"
+	"time"
 )
 
 const PrometheusAlertURL string = "http://localhost:9090/api/v1/alerts"
 
-func GetAlerts() ([]string, []byte) {
-	alertNames := make([]string, 0)
+var volumeArwState = []string{"\"disable-in-progress\"", "\"disabled\"", "\"dry-run\"", "\"dry-run-paused\"", "\"enable-paused\"", "\"enabled\""}
```

Comment:
bit easier to read like this?
volumeArwState = []string{
`"disable-in-progress"`,
`"disabled"`,
`"dry-run"`,
`"dry-run-paused"`,
`"enable-paused"`,
`"enabled"`,
}